gosnmp.go

```diff
@@ -111,10 +111,6 @@ type GoSNMP struct {
 	// See comments in https://github.com/gosnmp/gosnmp/issues/100
 	MaxRepetitions uint32
 
-	// NonRepeaters sets the GETBULK max-repeaters used by BulkWalk*.
-	// (default: 0 as per RFC 1905)
```

Comment:
Rather than fully remove this, can you mark it `Deprecated: This parameter is not used and ignored` per Go syntax conventions?

We can keep the behavior change in `walk.go`, but keeping the struct field avoids breaking the API.